### DISCUSSION TITLE
2.5i in hand!

### DISCUSSION THREAD

**Discussion by mikekay1** [Kaldek/rat-ratgdo#94] [General]
Hey Kaldek,

Mike here, as promised came back with 2.5i.

Is there anything I can test for everyone?

**Comment by rlowens** [comment]
First, can you verify what it says on each chip? Maybe post some nice close-ups of each?

Closest we have so far is the order page pic:
![image](https://github.com/Kaldek/rat-ratgdo/assets/35885181/443b4809-d167-4784-901c-acd98dafd96e)

**Comment by judeibe** [comment]
![image](https://github.com/Kaldek/rat-ratgdo/assets/8676/c2b55d9f-fb88-4cab-8601-dc46263136db)
![image](https://github.com/Kaldek/rat-ratgdo/assets/8676/15a7e5d8-ed71-4a0a-ad93-6a16e414566c)
![image](https://github.com/Kaldek/rat-ratgdo/assets/8676/c5e601ee-f06a-4093-a0bf-965d8fa737cf)
![image](https://github.com/Kaldek/rat-ratgdo/assets/8676/6b2bc8b6-4091-45e6-a5f8-3754c97425d2)
![image](https://github.com/Kaldek/rat-ratgdo/assets/8676/6be2291e-329f-4cc2-ab1f-5b9c0121575e)

**Comment by mikekay1** [comment]
Ah sorry guys, didn't get to it, looks like someone beat me to it! I'll post mine for comparison.
![20240118_184848](https://github.com/Kaldek/rat-ratgdo/assets/52670539/0e095f6a-8a6a-4a22-9b48-78c451f66ab2)
![20240118_184842](https://github.com/Kaldek/rat-ratgdo/assets/52670539/82b17103-aab2-4d70-aa8b-7266b7fe1005)
![20240118_184836](https://github.com/Kaldek/rat-ratgdo/assets/52670539/79e53011-5a49-4cd6-a011-31fbd7e67c92)
![20240118_184825](https://github.com/Kaldek/rat-ratgdo/assets/52670539/005c53c8-00cd-47d9-905c-250376b5708c)
![20240118_184818](https://github.com/Kaldek/rat-ratgdo/assets/52670539/315dad8a-c6ff-4894-9849-1e0e152598cf)
